google-beta/resource_pubsub_subscription.go

```diff
@@ -16,42 +34,38 @@ func resourcePubsubSubscription() *schema.Resource {
 		Delete: resourcePubsubSubscriptionDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: resourcePubsubSubscriptionStateImporter,
+			State: resourcePubsubSubscriptionImport,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(240 * time.Second),
+			Update: schema.DefaultTimeout(240 * time.Second),
+			Delete: schema.DefaultTimeout(240 * time.Second),
 		},
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: comparePubsubSubscriptionBasename,
```

Comment:
Same as for `topic`; I'd rather reuse `compareSelfLinkOrResourceName`.

Reply:
Let's keep the discussion under `topic` above.